src/api/ddoc/views.rst

```diff
@@ -157,13 +157,10 @@
     transfer size for attachments.
 
 .. http:post:: /{db}/_design/{ddoc}/_view/{view}
-    :synopsis: Returns certain rows for the specified stored view
+    :synopsis: Returns results for the specified stored view
 
     Executes the specified view function from the specified design document.
-    Unlike :get:`/{db}/_design/{ddoc}/_view/{view}` for accessing views, the
-    :method:`POST` method supports the specification
-    of explicit keys to be retrieved from the view results. The remainder of
-    the :method:`POST` view functionality is identical to the
+    :method:`POST` view functionality is identical to the
```

Comment:
This is a bit misleading. I'd reword this to be something along the lines of:

```
:method:`POST` view functionality supports identical parameters and behavior
as specified in :get:`/{db}/_design/{ddoc}/_view/{view}` API but allows for the
query string parameters to be supplied as keys in JSON object in the body
of the `POST` request.
```

Reply:
Good call! I'll push a commit for this soon.

Reply:
Fixed in 3cf6106a6a320c1b3f35ba29cd96c1def42db84d